### ISSUE TITLE
Add fMRIprep 22.0.1

### ISSUE THREAD

**Issue by jsmentch** [ReproNim/containers#72]
A new fMRIprep patch was just released that some folks would like to use for projects - can this new image be added?

**Comment by yarikoptic** [comment]
I believe Cron job runs daily, so if was just released, might take a bit to appear here. I will adjust to run hourly - should be cheap if nothing to do

**Comment by jsmentch** [comment]
Cool, didn't realize it was automated in this way or that all of the patch versions would get added here.

It was released 2 days ago so hopefully the job didn't miss it somehow

**Comment by yarikoptic** [comment]
Thanks again!  
- indeed the run was not "clean" but didn't crash our "resilient" bash script. Fixed that part now in bd301dd182fe04352244288f7aab51e3d88eab71 -- it does crash now ;)
- bad news: apparently the entire end-point we use https://registry.hub.docker.com/v1/repositories/ is `410 - Gone` now!!
- good news: there is `/v2` and it is pretty much identical. Fix is in d79d125b63d87b7560556d315c83df629645da31 , running the script now.... it is working on bids-hippunfold--1.2.0 ATM.
- checked: cron job runs each hour

I will come back and close or follow up whenever the run finishes up.

**Comment by yarikoptic** [comment]
done!  I bet there are other gotchas ("out of cheese"), I `git reset --hard` few steps back - hopefully no side effects, but should be good for now.

- oddity/surprise -- both versions are EXACTLY the same size image which made me worry but seems to be ok
```
(git)smaug:~/proj/repronim/containers[master]
$> singularity run images/bids/bids-fmriprep--22.0.1.sing --version
fMRIPrep v22.0.1
singularity run images/bids/bids-fmriprep--22.0.1.sing --version  6.05s user 0.94s system 94% cpu 7.421 total
2 8726.....................................:Thu 15 Sep 2022 04:44:38 PM EDT:.
(git)smaug:~/proj/repronim/containers[master]
$> singularity run images/bids/bids-fmriprep--22.0.0.sing --version
You are using fMRIPrep-22.0.0, and a newer version of fMRIPrep is available: 22.0.1.
Please check out our documentation about how and when to upgrade:
https://fmriprep.readthedocs.io/en/latest/faq.html#upgrading
fMRIPrep v22.0.0
singularity run images/bids/bids-fmriprep--22.0.0.sing --version  6.04s user 0.93s system 95% cpu 7.325 total
2 8727.....................................:Thu 15 Sep 2022 04:44:48 PM EDT:.
(git)smaug:~/proj/repronim/containers[master]
$> ls -lL images/bids/bids-fmriprep--22.0.[01].sing
-r-xr-xr-x 1 yoh yoh 3724525599 Jul 28 17:13 images/bids/bids-fmriprep--22.0.0.sing*
-r-x------ 1 yoh yoh 3724525599 Sep 15 16:40 images/bids/bids-fmriprep--22.0.1.sing*
```

- "out of cheese" - ran out of space on dedicated /mnt/docker , which silently lead to half brewed/broken singularity image and it seems that process finished just ok... not good... need to figure it out
- something odd with pushing -- permissions... uff...

overall this specific issue should be fixed as of https://github.com/ReproNim/containers/commit/c5c36991fc58a0ec108ed119c481ff069cef3a9a a few minutes ago
